account-kit/smart-contracts/src/ma-v2/client/client.test.ts

```diff
@@ -7,12 +18,16 @@ import { accounts } from "~test/constants.js";
 import { getDefaultSingleSignerValidationModuleAddress } from "../modules/utils.js";
 import { SingleSignerValidationModule } from "../modules/single-signer-validation/module.js";
 import { installValidationActions } from "../actions/install-validation/installValidation.js";
+import { semiModularAccountBytecodeAbi } from "../abis/semiModularAccountBytecodeAbi.js";
 
 describe("MA v2 Tests", async () => {
   const instance = local070Instance;
   let client: ReturnType<typeof instance.getClient> &
     ReturnType<typeof publicActions>;
 
+  const isValidSigSuccess = "0x1626ba7e";
+  const isValidSigFail = "0xffffffff";
```

Comment:
⚠️ **[eslint]** <[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'isValidSigFail' is assigned a value but never used.
<!-- __reviewdog__:ChA3YTg5N2U0MGNlZGUzZWVmEgZyZGpzb24= -->